File: Estrutura_Condicional/clojure/Ex1048.clj
(let [salario (read)
      ajustar-salario (fn [salario-original percentual]
                        (printf (str "Novo salario: %.2f\n"
                                     "Reajuste ganho: %.2f\n"
                                     "Em percentual: %.0f %%\n")
                                (+ salario-original (* salario-original percentual))
                                (* salario-original percentual)
                                (* 100 percentual)))]
  (cond
    (> salario 2000.00) (ajustar-salario salario 0.04)
    (>= salario 1200.01) (ajustar-salario salario 0.07)
    (>= salario 800.01) (ajustar-salario salario 0.10)
    (>= salario 400.01) (ajustar-salario salario 0.12)
    (>= salario 0.0) (ajustar-salario salario 0.15)))

File: Estrutura_Condicional/Python/Ex1048.py
salario = float(input())
salarioF = 0
reajuste = 0

if 0 <= salario <= 400:
    reajuste = salario * 0.15
    salarioF = salario + reajuste
    print(f"Novo salario: {salarioF:.2f}")
    print(f"Reajuste ganho: {reajuste:.2f}")
    print("Em percentual: 15 %")
elif salario <= 800:
    reajuste = salario * 0.12
    salarioF = salario + reajuste
    print(f"Novo salario: {salarioF:.2f}")
    print(f"Reajuste ganho: {reajuste:.2f}")
    print("Em percentual: 12 %")
elif salario <= 1200:
    reajuste = salario * 0.10
    salarioF = salario + reajuste
    print(f"Novo salario: {salarioF:.2f}")
    print(f"Reajuste ganho: {reajuste:.2f}")
    print("Em percentual: 10 %")
elif salario <= 2000:
    reajuste = salario * 0.07
    salarioF = salario + reajuste
    print(f"Novo salario: {salarioF:.2f}")
    print(f"Reajuste ganho: {reajuste:.2f}")
    print("Em percentual: 7 %")
else:
    reajuste = salario * 0.04
    salarioF = salario + reajuste
    print(f"Novo salario: {salarioF:.2f}")
    print(f"Reajuste ganho: {reajuste:.2f}")
    print("Em percentual: 4 %")